dags/config/gcs.yaml

```diff
@@ -14,6 +14,9 @@ source:
       file_prefix_filtering: False
       delete_gcs_files: False
       sensor_timeout: 6000
+      airflow_date_template: "ds"
```

Comment:
wondering if it makes sense to make these default value, and then make them optional in the config

Reply:
In addition, date_format also didn't include by default which causes an exception.